telco-core/configuration/reference-crs-kube-compare/required/networking/metallb/bfd-profile.yaml

```diff
@@ -12,7 +12,7 @@ spec:
   echoInterval: {{ if .spec.echoInterval }} {{ .spec.echoInterval }} {{ else }} "50ms" {{ end }} # default 50ms
   {{ end }}
   detectMultiplier: {{ if .spec.detectMultiplier }} {{ .spec.detectMultiplier }} {{ else }} "3" {{ end }} # default 3
-  echoMode: true
+  echoMode: {{ if .spec.echoMode }} {{ .spec.echoMode }} {{ end }}
```

Comment:
I think this will end up with unparseable yaml if .spec.echoMode is not present.  Maybe this would work better:

```suggestion
  {{- if .spec.echoMode }}
  echoMode: {{ .spec.echoMode }}
  {{- end }}
```

Reply:
thanks, updated